frontend/src/Components/RolesAndRules.js

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import styles from '../Styles/RolesAndRules.module.css';
+import { Paper } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles({
+    root: {
+        padding: '1em',
+        display: 'grid',
+        gridTemplateRows: '2em 1fr 0.5fr',
+        alignItems: 'stretch',
+        justifySelf: 'end',
+    },
+    good: { backgroundColor: 'palegreen' },
+    bad: { backgroundColor: 'lightcoral' },
+});
+
+/**
+ * @param userRole Civilian/Medic/Detective/Mafia - gets user's role to display what their current role is
+ */
+const RolesAndRules = ({ userRole }) => {
```

Comment:
Not really anything wrong but are we keeping to the files being .jsx rather than js?

Reply:
For consistency's sake it's probably a good idea to. #93 could be used to do this. We'll also get around to it when doing a code cleanup later on.